projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/prompt/index.ts

```diff
@@ -104,6 +104,87 @@ ${ postTitle?.length ? `- Current title: ${ postTitle }\n` : '' }${
 	};
 }
 
+type PromptOptionsProps = {
+	content: string;
+	language?: string;
+	tone?: ToneProp;
+	role?: PromptItemProps[ 'role' ];
+};
+
+function getCorrectSpellingPrompt( {
+	content,
+	role = 'user',
+}: PromptOptionsProps ): Array< PromptItemProps > {
+	return [
+		{
+			role,
+			content: `Repeat the following text, correcting any spelling and grammar mistakes:\n\n${ content }`,
+		},
+	];
+}
+
+function getSimplifyPrompt( {
+	content,
+	role = 'user',
+}: PromptOptionsProps ): Array< PromptItemProps > {
+	return [
+		{
+			role,
+			content: `Simplify the following text, using words and phrases that are easier to understand. Write the content in the same language as the original content:\n\n${ content }`,
+		},
+	];
+}
+
+function getSummarizePrompt( {
+	content,
+	role = 'user',
+}: PromptOptionsProps ): Array< PromptItemProps > {
+	return [
+		{
+			role,
+			content: `Summarize the following text. Write the content in the same language as the original content:\n\n${ content }`,
+		},
+	];
+}
+
+function getExpandPrompt( {
+	content,
+	role = 'user',
+}: PromptOptionsProps ): Array< PromptItemProps > {
+	return [
+		{
+			role,
+			content: `Expand the following text to about double its size approximately. Write the content in the same language as the original content:\n\n${ content }`,
```

Comment:
```suggestion
			content: `Expand the following text to about double its size. Write the content in the same language as the original content:\n\n${ content }`,
```
The "about" should be enough for the intended meaning.